charmcraft/application/commands/init.py

```diff
@@ -113,6 +115,10 @@ def _make_success_message(src_files: list[str]) -> str:
 {src_files_str}
 README.md
 """
+    if profile in ("kubernetes", "machine"):
+        message = f"""{message}
```

Comment:
Should this be `+=`?
```suggestion
        message += f"""{message}
```

Reply:
My mistake, I missed that `message` was included in the new message via f-string. Either way is fine, imo -- the current code is fine.